node_modules/gh-events/lib/api.js

```diff
@@ -300,6 +300,87 @@ var updateEvent = module.exports.updateEvent = function(ctx, id, update, callbac
 };
 
 /**
+ * Update the organisers of an event
+ *
+ * @param  {Context}        ctx                     Standard context containing the current user and the current app
+ * @param  {Number}         id                      The id of the event to update
+ * @param  {Object}         update                  The updates to persist. Keys that are numbers are considered to be references to users, anything else is considered to be a plain-text organiser
+ * @param  {Function}       callback                Standard callback function
+ * @param  {Object}         callback.err            An error object, if any
+ * @param  {Event}          callback.event          The updated event
+ */
+var updateEventOrganisers = module.exports.updateEventOrganisers = function(ctx, id, update, callback) {
+    update = update || {};
+
+    var validator = new Validator();
+    validator.check(null, {'code': 401, 'msg': 'Only authenticated users can update events'}).isLoggedInUser(ctx);
+    validator.check(id, {'code': 400, 'msg': 'A valid event id must be provided'}).isInt();
+    validator.check(_.keys(update).length, {'code': 400, 'msg': 'At least 1 update value must be provided'}).min(1);
+    _.each(update, function(val, user) {
+        update[user] = GrasshopperUtil.getBooleanParam(update[user]);
+        validator.check(null, {'code': 400, 'msg': 'A valid organiser change must be a boolean value'}).isBoolean(update[user]);
+    });
+    if (validator.hasErrors()) {
+        return callback(validator.getFirstError());
+    }
+
+    // Ensure the event exists
+    getEvent(ctx, id, function(err, event) {
+        if (err) {
+            return callback(err);
+        }
+
+        // Get the user ids, if any, from the update
+        var userIds = _.chain(update)
+            .keys()
+            .map(function(user) {
+                return parseInt(user, 10);
+            })
+            .filter(function(val) {
+                // Note that we cannot use _.isNumber here as NaN is considered to be a Number too
+                // See https://lodash.com/docs#isNumber
+                return (!_.isNaN(val));
+            })
+            .value();
+
+        // Ensure the specified users exist, if any
+        _validateOptionalUsers(ctx, userIds, event.AppId, function(err, users) {
+            if (err) {
+                return callback(err);
+            }
+
+            // Ensure that the current user is allowed to update the event
+            EventsAuthz.canUpdateEvent(ctx, event, function(err, canUpdateEvent) {
+                if (err) {
+                    return callback(err);
+                } else if (!canUpdateEvent) {
+                    return callback({'code': 401, 'msg': 'You are not allowed to update this event'});
+                }
+
+                // TODO: Add a "confirm"-flow for the organisers, so regular users can't add other
```

Comment:
I don't think we need this at the moment. We can add it when a production need arises